.drone.star

```diff
@@ -4,7 +4,8 @@ OC_UBUNTU = "owncloud/ubuntu:20.04"
 PLUGINS_GITHUB_RELEASE = "plugins/github-release:1"
 POSTGRESQL = "postgres:13"
 MOODLEHQ_APACHE = "moodlehq/moodle-php-apache:8.1"
-OC_OCIS = "owncloud/ocis:5.0.0-rc.5"
+OC_CI_GOLANG = "owncloudci/golang:1.21"
```

Comment:
how can that work? on master its 1.22.0 https://github.com/owncloud/ocis/blob/master/go.mod#L3